__tests__/filename.spec.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: Ferdinand Thiessen <opensource@fthiessen.de>
+ * SPDX-License-Identifier: AGPL-3.0-or-later or LGPL-3.0-or-later
+ */
+
+import { describe, it, expect, beforeEach, vi } from 'vitest'
+
+describe('isFilenameValid', () => {
+	beforeEach(() => {
+		delete window._oc_config
+		vi.resetModules()
+	})
+
+	it('works for valid filenames', async () => {
+		const { isFilenameValid } = await import('../lib/index')
```

Comment:
Why no global import? 🤔

Reply:
We need to reset the module for every test to reset the cached forbidden characters

Reply:
Doesn't Vitest have reset module option?

Reply:
With `vi.resetModules()`

Reply:
> Doesn't Vitest have reset module option?

Yes see rest of the file, but it can only reset the node cache. So top level imports do not work. It only resets the module cache -> next import will be freshly evaluated.

Reply:
Why are we always on edge cases :see_no_evil: :sob: